src/StatusQ/Components/StatusTagSelector.qml

```diff
@@ -139,9 +140,9 @@ Item {
             for (var i = 0; i < inputModel.count; i++ ) {
                 var entry = inputModel.get(i);
                 if (entry.name.toLowerCase().includes(text.toLowerCase())) {
-                    sortedList.insert(sortedList.count, {"publicId": entry.publicId, "name": entry.name,
-                                          "icon": entry.icon, "isIdenticon": entry.isIdenticon,
-                                          "onlineStatus": entry.onlineStatus});
+                    sortedList.append({"publicId": entry.publicId, "name": entry.name,
```

Comment:
Curious why this hasn't been an issue before...

Reply:
This line does not cause any problems. I changed this during researching the problem. append() is a good function to append :)